src/ansys/sherlock/core/errors.py

```diff
@@ -1121,7 +1121,7 @@ def str_itr(self):
         return [f"Update test fixtures by file error: {self.message}"]
 
 
-class SherlockExportAllTestPoints(Exception):
+class SherlockExportAllTestPointsError(Exception):
```

Comment:
Since the exportAllTestPoints API returns a ReturnCode message, this class will never need the error_array. This class should be modified to only support a single error message.